### DISCUSSION TITLE
Clear/Run a single Task instance using API Version 2.0.1 - clearTaskInstances

### DISCUSSION THREAD

**Discussion by Sanaj2060** [apache/airflow#29651] [Q&A]
I'm trying to rerun a already success task instance by using the "clearTaskInstances"  method in API Version 2.0.1. 

The API call return:
{
  "detail": "{'task_ids': ['Unknown field.']}",
  "status": 400,
  "title": "Bad Request",
  "type": "https://airflow.apache.org/docs/2.0.1/stable-rest-api-ref.html#section/Errors/BadRequest"
}

The request worked fine without task_ids, but I need to provide specific task id to be cleared.

Doc ref: [here](https://airflow.apache.org/docs/apache-airflow/2.1.0/stable-rest-api-ref.html#operation/post_clear_task_instances)

**Comment by potiuk** [comment]
I believe it was not there in https://airflow.apache.org/docs/apache-airflow/2.0.1/stable-rest-api-ref.html#operation/post_clear_task_instances - it was added later - you can take a look at release notes when. I recommend you to upgrade to latest Airflow version anyway. It's always recommended.
